extensions/elytron-security-oauth2/deployment/src/main/java/io/quarkus/elytron/security/oath2/deployment/OAuth2DeploymentProcessor.java

```diff
@@ -0,0 +1,138 @@
+package io.quarkus.elytron.security.oath2.deployment;
```

Comment:
The package name should be `io.quarkus.elytron.security.oauth2.deployment` instead ? Similar to what is used in runtime module.

Reply:
Small typo, done